docs/pages/build/orbit.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Expo Orbit
+maxHeadingDepth: 4
+description: Accelerate your development workflow with one-click build launches and simulator management.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import Video from '~/components/plugins/Video';
+
+Expo Orbit for macOS makes it faster and easier to install and run builds from EAS or elsewhere, and to run Snack projects on simulators and physical devices.
+
+<Video file="orbit/basic-features.mp4" />
+
+The process for installing builds from EAS (Android device/emulator, iOS simulator, iOS device) and running Snack projects on simulators used to be more manual than it needs to be. You could use `eas build:run` and select a build for Android devices/emulators or iOS simulators, or you could download the archive, extract it, and drag and drop it to the simulator/emulator. For Snack projects, additional steps include installing Expo Go in the simulator through the CLI, logging in, and selecting the Snack from a list. To improve this we released Orbit, making these steps as seamless as possible and aligning with the user-friendly experience that Expo offers
+
+## Highlights
+
+- List and launch simulators, including the ability to run Android emulators without audio
+- Install and launch builds from EAS to your simulators and real devices with just one click
+- Launch Snack projects in your simulators by clicking a button
+- Install and launch apps from local files using Finder or just drag and drop a file into the menu bar app. Orbit supports any Android apk, simulator compatible .app, or ad-hoc signed apps
```

Comment:
```suggestion
- Install and launch apps from local files using Finder or just drag and drop a file into the menu bar app. Orbit supports any Android APK, simulator compatible **.app**, or ad hoc signed apps
```

canonical spelling for ad-hoc as per apple is ad hoc - https://developer.apple.com/help/account/manage-profiles/create-an-ad-hoc-provisioning-profile/

Reply:
makes sense, thanks for fixing this